app/models/manifest.rb

```diff
@@ -29,6 +31,12 @@ def initialize(path, opts)
     @name = opts.fetch("name", default_name)
     @host = opts.fetch("host", default_host)
     @notes = opts.fetch("notes", default_notes)
+
+    if !!opts.fetch("notes_is_markdown", false)
```

Comment:
I would just render all of the text as markdown and not check for a boolean "notes_is_markdown". 

The descriptions are really simple right now: plain text. It is a change, we can just document it well. Perhaps we should rescue an exception and display the plain text if the exception occurs. Because for sure we souldn't crash just because the markdown is badly formatted. But my guess is most people will be pleased when their links in their descriptions suddenly become clickable.

If we have this be "opt-in" then most people will not benefit from this. Also I would find it annoying to have to change our 50 or so templates to include `notes_is_markdown: true` in the YAML.